docs/agent-vs-aea.md

```diff
@@ -6,7 +6,15 @@ In this guide we show some of the differences in terms of code.
 
 The <a href="../build-aea-programmatically">Build an AEA programmatically</a> guide shows how to programmatically build an AEA. We can build an agent of the <a href="../api/agent#agent-objects">`Agent`</a> class programmatically as well.
 
-First, import the python and application specific libraries. (Get the packages directory from the AEA repository `svn export https://github.com/valory-xyz/open-aea.git/trunk/packages`.)
+First, create an empty agent and add the stub connection and default protocol.
+```bash
+aea create my_aea
+cd my_aea
+aea add connection fetchai/stub:0.21.0:QmektTWmXcjThQd8md8nSYgLapR3Gks3n3WEzwAWQFgc4z --remote
+aea add protocols fetchai/default:1.0.0:QmYNdsSrdKRvJGKjAbREuvkjGXgnanDjxCBS8CfJb9fzr1 --remote
+```
```

Comment:
New approach: do it as a new agent with the required packages instead of checking the packages folder.

Reply:
You can then add:
```
aea push ... --local
aea push ... --local
cd ..
aea delete my_aea
```
for both packages. This will mean there's then a packages folder with the required packages :)